apps/api/index.d.ts

```diff
@@ -1,3 +1,5 @@
 import * as Schema from './prisma/schema';
 
 export { Schema };
+
+export type CaseType = 'household' | 'full planning' | 'application';
```

Comment:
hmm I have seen this a few places, but thought I should ask it now.  The type of appeal is a "Householder Appeal" not "Household Appeal" - so should this say householder, not household?

Reply:
does this need confirmation

Reply:
That's interesting :D no one said anything before. I am a little on the fence about making changes to the appeal side of the service

Reply:
cool - let's leave it as is, when we work on Appeals again and this is raise as a requirement it can be changed